contracts/components/Claims.sol

```diff
@@ -133,14 +133,13 @@ contract Claims {
     event Dispute(
         IArbitrator indexed _arbitrator,
         uint256 indexed _disputeID,
-        uint256 _metaEvidenceID,
-        uint256 _evidenceGroupID
+        uint256 _metaEvidenceID
     );
 
     event CreateClaim(
         address indexed createdBy,
-        string indexed safeId,
```

Comment:
I think we should keep the `safeId` but remove `createdBy`

Reply:
Done